joint_limits/include/joint_limits/joint_limits_rosparam.hpp

```diff
@@ -58,7 +125,7 @@ namespace joint_limits_interface
  * existing values. Values in \p limits not specified in the parameter server remain unchanged.
```

Comment:
The comments above this could use updating:

> This specification is similar to the one used by <a href="http://moveit.ros.org/wiki/MoveIt!">MoveIt!</a>, but additionally supports effort limits.

(MoveIt handles jerk limits now.)

Also needs soft limits information to be added, I think.